src/Model/Ticket/SlaPlan.php

```diff
@@ -206,4 +212,23 @@ public function setName(string $name): self
 
         return $this;
     }
+
+    /**
+     * @return SlaPlanTranslation[]|null
+     */
+    public function getTranslations(): ?array
+    {
+        return $this->translations;
+    }
+
+    /**
+     * @param SlaPlanTranslation[]|null $translations
+     * @return self
+     */
+    public function setTranslations(?array $translations): self
```

Comment:
Are the types for this and the get function correct?

Reply:
Should be, it's an array of translations

Did you spot any issues?